src/Components/PresentationalComponents/Kebab/KebabItems.js

```diff
@@ -3,22 +3,14 @@ import React from 'react';
 import messages from '../../../Messages';
 import { FormattedMessage } from 'react-intl';
 
-export const kebabItemDownloadJSON = (downloadReport, { ...props }) => (
-    <DropdownItem key="json" component="button" onClick={() => downloadReport('json')} {...props}>
-        <FormattedMessage {...messages.kebabExportAsJson} />
-    </DropdownItem>
-);
-
-export const kebabItemDownloadCSV = (downloadReport, { ...props }) => (
-    <DropdownItem key="csv" component="button" onClick={() => downloadReport('csv')} {...props}>
-        <FormattedMessage {...messages.kebabExportAsCsv} />
-    </DropdownItem>
-);
-
+// Workaround: Using button instead of DropdownItem as it causes error on Dropdown open
+// in the current version of Patternfly
 export const kebabItemDownloadPDF = (loading, downloadReport, { ...props }) => (
-    <DropdownItem key="pdf" component="button" onClick={() => downloadReport(true)} {...props}>
-        {loading ? <FormattedMessage {...messages.loading} /> : <FormattedMessage {...messages.kebabExportAsPDF} />}
-    </DropdownItem>
+    <li>
+        <button id="kebab-item-download-pdf" key="pdf" component="button" onClick={() => downloadReport(true)} {...props}>
+            {loading ? <FormattedMessage {...messages.loading} /> : <FormattedMessage {...messages.kebabExportAsPDF} />}
```

Comment:
A nit-pick: the behavior of the other two buttons (export in CSV, JSON) is somewhat different: once clicked, the toolbar is closed, and they don't change its inner string to "Loading..." Is it desired?

Reply:
Yes, this was the previous behaviour, it is because PDF export takes much more time, dropdown is not closed so the user can see the `Loading...` message.